test.sh

```diff
@@ -18,18 +23,66 @@ function main() {
   runHTTPSTests
```

Comment:
`finish` should probably be called at the end of this to tear down the OSS containers